docs/src/main/tut/kleisli.md

```diff
@@ -136,6 +137,9 @@ implicit def kleisliFlatMap[F[_], Z](implicit F: FlatMap[F]): FlatMap[Kleisli[F,
 
     def map[A, B](fa: Kleisli[F, Z, A])(f: A => B): Kleisli[F, Z, B] =
       Kleisli(z => fa.run(z).map(f))
+
+    def tailRecM[A, B](a: A)(f: A => Kleisli[F, Z, A Xor B]) =
+      Kleisli[F, Z, B]({ z => FlatMap[F].tailRecM(a) { f(_).run(z) } })
```

Comment:
We should probably write a note somewhere for these tuts why we also have `tailRecM` on monad instances. I've added this ticket https://github.com/typelevel/cats/issues/1284 to track it